modules/hcp-ibm-power-heterogeneous-nodepools-agent-hc-dns.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-ibm-power-heterogeneous-nodepools-agent-hc-dns_{context}"]
+= Configuring DNS
+
+You can point `*.apps.<cluster_name>` record to either one of the worker nodes where ingress application is hosted, or if you are able to set up a load balancer on top of the worker nodes, point this record to this load balancer. When you are creating a heterogeneous nodepool, make sure the workers are reachable from each other or keep them in the same network.
```

Comment:
What is *.apps.<cluster_name>? When would I do this? Is it required?

Reply:
```suggestion
You can point an `*.apps.<cluster_name>` record to either one of the worker nodes where ingress application is hosted. Or, if you are able to set up a load balancer on top of the worker nodes, point this record to this load balancer. When you are creating a heterogeneous nodepool, make sure the workers are reachable from each other or keep them in the same network.
```

Reply:
> What is *.apps.<cluster_name>? When would I do this? Is it required?

It is a wildcard DNS A/AAAA or CNAME record that targets the node. This record must be resolvable by clients external to the cluster. Reference: https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/nodes/worker-nodes-for-single-node-openshift-clusters